plugins/node/instrumentation-tedious/README.md

```diff
@@ -17,7 +17,7 @@ npm install --save @opentelemetry/instrumentation-tedious
 
 ## Supported Versions
 
-- `>=1.11.0 <=15`
+- `>=1.11.0 <=19`
```

Comment:
_issue_: 
I think this needs to be `<=18` or `<19`
Latest version of tedious is 18.2.0

Reply:
right! my bad

Reply:
changed to `<19`